### ISSUE TITLE
[CAIA A11y Research]: Veteran-facing-forms, Form Header Usability Testing (Oct/Nov 2023)

### ISSUE THREAD

**Issue by PSIEBENLIST** [department-of-veterans-affairs/va.gov-team#65316]
## CAIA Assignees
**Support primary goal:** assist users with assistive technology, should any issues take place, during the sessions.
**Additionally:** Identify and document any accessibility-related issues encountered in the user testing sessions. 

- [x] Sara
- [x] SK @SarahKay8 
- [x] Jamie @coforma-jamie 
- [x] **`RCVD 9/20` session times for dates provided:** September 25 - October 5, 2023. 
There are currently ~three~ no studies overlapping during this timeframe, and we will do our very best to provide support for your AT sessions.
- [x] Added date range to the [CAIA Capacity Calendar](https://calendar.google.com/calendar/embed?height=600&wkst=1&bgcolor=%23ffffff&ctz=America%2FNew_York&mode=WEEK&src=Y19mOTgyMzUwZmM3M2E4MTYyYTEwODhhYjE0ZDAwZGQzNzMzNWIyYjU1OWE2MGQ2YjY3ZWI4MDIxNDE2ZjgxYzJhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23616161).

### Who's on your team?

```bash
Team name: Veteran-facing-forms
OCTO-DE product owner: Matthew Dingee
Product manager: Pat Siebenlist
Designer: Rachel Shearer 
FE engineer: Robert Hasselle
Product/team Slack channel: #veteran-facing-forms
...
```

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

September 25 - October 5

### Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2023-09-form-header-usability-testing/Product%20Outline.md)
- [User flows](https://staging.va.gov/authorization-to-disclose-alternate/introduction)
- [Prototype or mockups](https://staging.va.gov/authorization-to-disclose-alternate/introduction)
- [Research plan](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2023-09-form-header-usability-testing/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/research/2023-09-form-header-usability-testing/conversation-guide.md)
- [Epics/issue that may be helpful](Insert URL here between the parenthesis)
...

```[tasklist]
### Next Steps
- [x] Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [x] CAIA Specialists update the CAIA Capacity Calendar with `HOLDS`
- [x] Your team's researchers and observers read through platform guidance on research with assistive technology users
- [x] `ZOOM INVITES RCVD` Holds removed from Capacity Calendar, invites received, copied to calendar, along with assignees
- [x] CAIA Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [x] CAIA Specialists update the CAIA Capacity Calendar and remove holds.
- [x] CAIA Specialists review your conversation guide and research plan
- [x] Provide specialists your participant details like skill level, operating system, assistive tech used, browser
- [x] Complete October 2023 User Testing
- [x] CAIA Specialists create a research notes template
- [x] CAIA Specialists meet to sync on Findings (async 10/30/2023; meeting on 10/31/2023) @coforma-jamie
- [x] CAIA Post research, specialists will link to [accessibility findings](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Veteran%20Facing%20Forms/2023-Oct-Veteran-Facing-Forms-Form-Header-accessibility-findings.md) in the CAIA Deliverables Folder and attach the markdown file for you to upload to your folder, since CAIA does not have access.  11/3/2023
```

**Comment by rachelshearerux** [comment]
Hi friends, 5 screen-reader sessions have been scheduled for the week of October 2nd. Thank you in advance for your support!

**Tuesday, October 3**
- 11am to 12:30pm EST

**Wednesday, October 4**
- 11am to 12:30pm EST

**Thursday, October 5**
- 9am to 10:30am EST
- 11am to 12:30pm EST
- 1pm to 2:30pm EST

**Comment by sara-amanda** [comment]
## CAIA AT Sessions

- **Tuesday, October 3**
    - 11am to 12:30pm EST 
        -  @sara-amanda available
- **Wednesday, October 4**
    - 11am to 12:30pm EST
        - @sara-amanda available
- **Thursday, October 5**
    - 9am to 10:30am EST
        - `CAIA OPEN SLOT AVAILABLE`
    - 11am to 12:30pm EST
        - `CAIA OPEN SLOT AVAILABLE`
    - 1pm to 2:30pm EST
        - `CAIA OPEN SLOT AVAILABLE`

## Additional Information Needed
**To best serve the users of VA.gov, we will need to collect the following information during each a11y research session:**
- Participant: Number Only
- Participant Testing Date/Time: 
- Skill level of user: Novice, Intermediate or Advanced
- Operating System: Mac or Windows
- Assistive Technology being used: Jaws, NVDA, Zoomtext, Dragon, etc.
- Internet Browser: Chrome, Firefox, etc.
- Device Type: Mobile, Desktop or Tablet

@coforma-terry @SarahKay8

**Comment by SarahKay8** [comment]
Hi @rachelshearerux! Sara's comment has been updated with my availability

**Comment by rachelshearerux** [comment]
Thank you, @SarahKay8! There are two more sessions that got scheduled today. I'm sending these over as they're being scheduled, but as soon as I have access to Perigean's Participant Roster and Schedule I'll fill in some of the participant detail gaps you have listed.

New sessions scheduled as of today:
**Monday, 10/2**
- 11am to 12:30pm EST
- 1pm to 2:30pm EST

**Comment by sara-amanda** [comment]
Hi @rachelshearerux I have updated the comment with the two new sessions. Thanks for sharing! I will add them to our capacity calendar as well and will watch for the invites. 👍

**Comment by rachelshearerux** [comment]
Hi @sara-amanda! Perigean added 3 more session today and has closed scheduling. Can you please add these to your capacity calendar? I'll start sharing calendar invitations with zoom links. Thank you! 

**Tuesday, October 3**
- 9am-10:30am EST
- 1pm-2:30pm EST

**Wednesday, October 4**
- 1pm to 2:30pm

**Comment by rachelshearerux** [comment]
> Hi @sara-amanda! Perigean added 3 more session today and has closed scheduling. Can you please add these to your capacity calendar? I'll start sharing calendar invitations with zoom links. Thank you!
> 
> **Tuesday, October 3**
> 
> * 9am-10:30am EST
> * 1pm-2:30pm EST
> 
> **Wednesday, October 4**
> 
> * 1pm to 2:30pm

Hi friends, just checking to make sure the three new sessions above are on your calendar. Thanks!

**Comment by SarahKay8** [comment]
@rachelshearerux invites received 🦋

**Comment by SarahKay8** [comment]
Actually my mistake. @rachelshearerux ill be missing those but will be at the sessions on 10/5

**Comment by rachelshearerux** [comment]
@SarahKay8 ok, thanks for letting me know! 

@sara-amanda and @coforma-jamie, I just added you both as optional to the three AT user session below. We appreciate your support so much! 

> Tuesday, October 3
> - 9am-10:30am EST
> - 1pm-2:30pm EST

> Wednesday, October 4
> - 1pm to 2:30pm

**Comment by sara-amanda** [comment]
## Session Invite Inquiry
- 10/2/2023: 1pm to 2:30pm EST
- @rachelshearerux -`FYI invite received for this session as of 10/2/2023 10:37 a.m.`

**Comment by rachelshearerux** [comment]
Hi friends, here are a few new sessions Perigean has scheduled for us, there may be more than this but I wanted to get these over to you asap. Thanks for continuing to support us, we appreciate y'all so much!   

**Tuesday, October 10:**

- [ ] 11:30am to 1pm EST
- [ ] 1:30pm to 3pm EST

**Thursday, October 12:**
- [ ] 11:30am to 1pm EST

**Friday, October 13:**
- [ ] 9am to 10:30am EST 

**Monday, October 16**
- [ ]  9am to 10:30am EST

**Comment by sara-amanda** [comment]
## CAIA Sessions Update
@rachelshearerux the new sessions sent over today have been updated [in the running comment in this ticket, located here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65316#issuecomment-1728277399), along with our CAIA Capacity Calendar @coforma-terry 

### Unavailable
We apologize, but there are a few sessions we are unable to cover.
- 10/10/2023 at 1:30 p.m.
- 10/11/2023 at 11:30 a.m. 

### Pending
The following sessions are pending coverage, due to limited availability at these times. 
- 10/12/2023 at 1:30 p.m.
- 10/13/2023 at 9 a.m.

cc: @SarahKay8 @coforma-jamie

**Comment by sara-amanda** [comment]
## Sessions Completed 
As of 10/10/2023 at 11:30 a.m.
1. P10 - SR Microsoft Extension 
2. P12 - Jaws
3. P13 & P17 - Fusion `(same person?) `
5. P19A - VoiceOver
6. P19B - TalkBack
7. P20 - ZoomText

## Scheduled Session Support
Sessions CAIA is available to support later this week and next, based on the schedule from 10/10/2023: 
- 10/12/2023 at 11:30 a.m. (SK)
- 10/16/2023 at 9 a.m. (SS)

## Potential Support (TBD) 
_Due to limited availability._
- 10/12/2023 at 1:30 p.m. 
- 10/13/2023 at 9 a.m.

**Comment by sara-amanda** [comment]
## CAIA A11Y Research Findings

Thank you for including CAIA A11Ys in your user research sessions for assistive technology support, and to collect valuable feedback from your users, as it relates to assistive technology. 

_We look forward to partnering with your team again in the future._ 

### About this Report
While CAIA cannot require you to complete any updates to your product, we can provide your team with recommendations to help facilitate a smooth process for your project. 

_**Note**: We have condensed the report view, so please be sure to toggle and expand the details in the report sections._ 

### Veterans' Voices
- Impactful quotes were collected during this study to provide user-specific feedback, related to the findings. 
- Hear what veterans have to say in the quotes that were selected for inclusion, in this report.

### View the Report
- [Link to findings in the CAIA Deliverables Folder for Veteran Facing Forms](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Veteran%20Facing%20Forms/2023-Oct-Veteran-Facing-Forms-Form-Header-accessibility-findings.md)
- **Copy of the markdown file:** 
    - _A copy is attached to this comment for your team to upload to your research space in GitHub._
        - [2023-Oct-Veteran-Facing-Forms-Form-Header-accessibility-findings-2.md](https://github.com/department-of-veterans-affairs/va.gov-team/files/13254475/2023-Oct-Veteran-Facing-Forms-Form-Header-accessibility-findings-2.md)

<hr>

@PSIEBENLIST @jeana-adhoc @rachelshearerux @SarahKay8 @coforma-terry @coforma-jamie
